_infra/helm/sample/values.yaml

```diff
@@ -2,8 +2,8 @@ env: minikube
 namespace: minikube
 
 image:
-  devRepo: eu.gcr.io/ons-rasrmbs-management
-  name: eu.gcr.io/ons-rasrmbs-management
+  devRepo: europe-west2-docker.pkg.dev/ons-ci-rmrasbs/images
+  name: europe-west2-docker.pkg.dev/ons-ci-rmrasbs/images
```

Comment:
Not for this PR as its the same across all our charts, but I've never fully understood `devRepo` vs `name` for these two.